doc/instructions.md

```diff
@@ -6,12 +6,35 @@ To install all the packages on ubuntu 18.04 LTS 64 bit, you should do the follow
 
   2. install robotpkg: follow [the robotpkg installation website](http://robotpkg.openrobots.org/debian.html).
 
-  3. install HPP: `sudo apt-get install robotpkg-hpp-tutorial robotpkg-osg-dae`
+  3. install HPP:
+
+    ```bash
+    #pyver=27
+    pyver=36
+    sudo apt-get install robotpkg-py${pyver}-hpp-manipulation-corba robotpkg-py${pyver}-qt5-hpp-gepetto-viewer
+    ```
 
   4. install (optionnal) extra packages for demonstrations:
-     `sudo apt-get install robotpkg-hpp-tutorial robotpkg-osg-dae ros-melodic-pr2-description robotpkg-hpp-environments robotpkg-romeo-description
 
-  5. setup your environment variables by adding the following lines to your `.bashrc`:
+    - Tutorials (binary release of the tutorial is only available for Python 2.7):
+
+      ```bash
+      sudo apt-get install robotpkg-py27-hpp-tutorial
```

Comment:
I wonder why there is no package robotpkg-py36-hpp-tutorial. Although the package installs python files.

Reply:
Because hpp-tutorial require some ROS packages, and ROS doesn't provide python 3 binaries: http://robotpkg.openrobots.org/rbulk/robotpkg/doc/py-hpp-tutorial/index.html#py36-hpp-tutorial-4.9.0

It's on my todolist to provide those Python 3 versions of the ROS packages we need with robotpkg binaries, but not done yet.

Reply:
I do not think hpp_tutorial depends on ROS package. You may have been misled by variable CATKIN_PACKAGE_SHARE_DESTINATION that has no relation with catkin.

Reply:
This looks totally true, thanks for the notice ! I'll try to fix that.

Reply:
I guess we had some dependencies of dependencies on some robots for some tests, but I can confirm that for the incoming release, all ROS dependencies can be removed.

Reply:
`robotpkg-py36-hpp-tutorial` is now available, and doesn't need any ROS package :)